scripts/rollup/build.js

```diff
@@ -305,7 +305,7 @@ function getPlugins(
     blacklistFBJS(),
     // Use Node resolution mechanism.
     resolve({
-      skip: externals,
```

Comment:
There is not such option in this plugin.
I already made an update here
https://github.com/facebook/react/pull/13321